test/blackbox_tests/blackbox_tests_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package blackbox_tests_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestTransparentProxy(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Blackbox tests Suite")
```

Comment:
I think "black box tests" everywhere should be something like "kuma-net tests" or "ebpf tests"

Reply:
I called them as I thought in the future we might want to write blackbox tests for some other (non network related) cases, but I agree, that we can change it in the future, so I renamed `blackbox_tests` to `blackbox_network_tests`. Wdyt about it?

Reply:
yeah that's clearer